src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,17 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
```

Comment:
does this actually require a regexp? Can we not use the standard `replace(".","")` given the output of `extname()`?